### ISSUE TITLE
https not able to capture

### ISSUE THREAD

**Issue by ghost** [bogdanKnabe1/NetRunner#9]
While trying to capture https app doesn't captures the data also stops working

**Comment by bogdanKnabe1** [comment]
Yes, https traffic this build cannot capture, only http, but about a crash, provide information, maybe I can immediately tell you what is wrong or fix it. Sorry about late answer.
